lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -96,6 +97,9 @@ class UsesOptimizedImages extends ByteEfficiencyAudit {
           continue;
         }
 
+        // If naturalHeight or naturalWidth are undefined, information is not valid, skip.
```

Comment:
nit
```suggestion
        // If naturalHeight or naturalWidth are falsy, information is not valid, skip.
```
They could be 0. Can you update the rest of the comments as well.